.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["env", "react"]
```

Comment:
This configures Babel to be able to transform your code, Parcel will use this file.